app/components/Snackbar.tsx

```diff
@@ -163,23 +163,23 @@ export function useSnackbar() {
   const [snackbarTimeout, setSnackbarTimeout] = useState<ReturnType<
     typeof setTimeout
   > | null>(null);
-  function openSnackbar(message: string | ReactNode) {
+  function openSnackbar(message: string | ReactNode, duration = DURATION) {
     context.dispatch({ type: ActionTypes.ShowSnackbar, message });
     setSnackbarTimeout(
       setTimeout(() => {
         close();
-      }, DURATION)
+      }, duration)
     );
   }
 
-  function showSnackbar(message: string | ReactNode) {
```

Comment:
Is this change okay, or was there a reason the snackbar didn't allow custom duration? Four seconds seemed a little fast...

Reply:
Totally good 👍